css/css-grid-1/reference/grid-support-grid-auto-columns-rows-002-ref.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
Nit: If you remove `<html>`, `<head>` and `<body>` in the test, please remove it here too.